### ISSUE TITLE
Bazel JDT LS extension doesn't support the command 'java.bazel.connectProcessStreamSocket'

### ISSUE THREAD

**Issue by vorburger** [salesforce/bazel-vscode-java#49]
@guw long time no speak! Very sorry I went a bit silent earlier this year (too busy).

I noticed that you have done much work here, and thought I would give it another go.

https://github.com/enola-dev/enola is the project I have tried to open with it.

Using the `v1.23.109211202` that's currently published as Preview, it's failing for me, with this on _Output_ for _Bazel:_ 

```
2023-09-29 19:06:47.887 [info] Bazel log server listening on port 46083
2023-09-29 19:07:08.100 [error] Failed to register port with BLS: Bazel JDT LS extension doesn't support the command 'java.bazel.connectProcessStreamSocket'.
```

and various errors on the _Language Support for Java,_ among which the following may be most interesting:

```java
[Error - 7:07:08 PM] Sep 29, 2023, 7:07:08 PM Problems occurred when invoking code from plug-in: "org.eclipse.jdt.ls.core".
Bazel JDT LS extension doesn't support the command 'java.bazel.connectProcessStreamSocket'.
java.lang.UnsupportedOperationException: Bazel JDT LS extension doesn't support the command 'java.bazel.connectProcessStreamSocket'.
	at com.salesforce.bazel.eclipse.jdtls.commands.BazelJdtLsDelegateCommandHandler.executeCommand(BazelJdtLsDelegateCommandHandler.java:76)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler$1.run(WorkspaceExecuteCommandHandler.java:230)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler.executeCommand(WorkspaceExecuteCommandHandler.java:220)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$4(JDTLanguageServer.java:613)
	at org.eclipse.jdt.ls.core.internal.BaseJDTLanguageServer.lambda$0(BaseJDTLanguageServer.java:87)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(Unknown Source)

(...)

[Error - 7:07:13 PM] Sep 29, 2023, 7:07:12 PM Error occured while building workspace. Details: 
 message: The import picocli cannot be resolved; code: 268435846; resource: /home/vorburger/git/github.com/vorburger/enola/cli/src/main/java/dev/enola/cli/CLI.java; line: 20

(...)

message: CommandLine cannot be resolved to a type; code: 16777218; resource: /home/vorburger/git/github.com/vorburger/enola/cli/src/main/java/dev/enola/cli/CLI.java; line: 29

INFO: -<- Done loading .configSourceProviders extension point -<-
[Error - 7:14:12 PM] Sep 29, 2023, 7:14:12 PM core/lib/src/test/java/dev/enola/core [in _main] does not exist
core/lib/src/test/java/dev/enola/core [in _main] does not exist
Java Model Exception: Error in Java Model (code 969): core/lib/src/test/java/dev/enola/core [in _main] does not exist
	at org.eclipse.jdt.internal.core.JavaElement.newNotPresentException(JavaElement.java:550)
	at org.eclipse.jdt.internal.core.PackageFragmentRoot.getUnderlyingResource(PackageFragmentRoot.java:761)
	at org.eclipse.jdt.internal.core.PackageFragment.getUnderlyingResource(PackageFragment.java:426)
	at org.eclipse.jdt.internal.core.Openable.getUnderlyingResource(Openable.java:349)
	at org.eclipse.jdt.internal.core.CompilationUnit.getUnderlyingResource(CompilationUnit.java:986)
	at org.eclipse.jdt.ls.core.internal.handlers.BaseDiagnosticsHandler.collectNonJavaProblems(BaseDiagnosticsHandler.java:149)
	at org.eclipse.jdt.ls.core.internal.handlers.BaseDiagnosticsHandler.endReporting(BaseDiagnosticsHandler.java:135)
	at org.eclipse.jdt.internal.core.ReconcileWorkingCopyOperation.reportProblems(ReconcileWorkingCopyOperation.java:148)
	at org.eclipse.jdt.internal.core.ReconcileWorkingCopyOperation.executeOperation(ReconcileWorkingCopyOperation.java:112)
	at org.eclipse.jdt.internal.core.JavaModelOperation.run(JavaModelOperation.java:740)
	at org.eclipse.jdt.internal.core.JavaModelOperation.runOperation(JavaModelOperation.java:806)
	at org.eclipse.jdt.internal.core.CompilationUnit.reconcile(CompilationUnit.java:1325)
	at org.eclipse.jdt.ls.core.internal.handlers.BaseDocumentLifeCycleHandler.publishDiagnostics(BaseDocumentLifeCycleHandler.java:320)
	at org.eclipse.jdt.ls.core.internal.handlers.BaseDocumentLifeCycleHandler.publishDiagnostics(BaseDocumentLifeCycleHandler.java:283)
	at org.eclipse.jdt.ls.core.internal.handlers.BaseDocumentLifeCycleHandler$PublishDiagnosticJob.run(BaseDocumentLifeCycleHandler.java:739)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

**Comment by nschmeller** [comment]
+1 that I'm also running into this issue. I'd love to provide any debugging support needed!

**Comment by guw** [comment]
I don't think it's the project. It looks like a problem with the plug-in and LS setup. To confirm, you have the latest version of the RedHat Java extension?

**Comment by guw** [comment]
Fixed with 
https://github.com/salesforce/bazel-eclipse/commit/2025b7872d160d5733b7652651e8cf4ca06790e3 (available as 1.23.110021004)
